python/lsst/afw/cameraGeom/cameraFactory.py

```diff
@@ -39,7 +39,7 @@ def makeDetector(detectorConfig, ampInfoCatalog, focalPlaneToPupil):
     return Detector(
         detectorConfig.name,
         detectorConfig.id,
-        detectorConfig.detectorType,
+        DetectorType(detectorConfig.detectorType),
```

Comment:
pybind11 is pickier about enums than SWIG. This change is needed because `DetectorConfig.detectorType` is an int, but the value wanted is a `DetectorType` enum.